dbms/src/Flash/Coprocessor/InterpreterUtils.cpp

```diff
@@ -120,7 +120,7 @@ void orderStreams(
                 order_descr,
                 settings.max_block_size,
                 limit,
-                settings.max_bytes_before_external_sort,
+                getAverageThreshold(settings.max_bytes_before_external_sort, std::min(max_streams, pipeline.streams.size())),
```

Comment:
I prefer
```
auto avg_threshold = getAverageThreshold(settings.max_bytes_before_external_sort, pipeline.streams.size());
stream = std::make_shared<MergeSortingBlockInputStream>(xxxx, avg_threshold);
```
The count of sort stream is `pipeline.streams.size()` at this time, not `min(max_streams, pipeline.streams.size())`, even if `max_streams < pipeline.streams.size()`. So we shoud use `pipeline.streams.size()` or not `std::min(max_streams, pipeline.streams.size())` here?

Reply:
Done